resources/timezones/map_data.txt

```diff
@@ -1972,7 +1991,7 @@
 95|Asia/Rangoon
 960|Indian/Maldives
 961|Asia/Beirut
-962|Asia/Amman
+962|Asia/Riyadh
```

Comment:
This seems wrong to me. 962 is Jordan, so I expected Amman rather than Riyadh here.